slash/react/src/Accordion/Header.tsx

```diff
@@ -25,8 +39,57 @@ const Header = ({ children, className, classModifier, id }: HeaderProps) => {
 
   return (
     <summary className={componentClassName} id={id}>
-      <h3 className="af-accordion__item-title">{children}</h3>
-      <span className="glyphicon glyphicon-menu-down" />
+      {type === "light" ? (
+        <Title heading="h3">
+          <Svg src={arrow} className="af-accordion__item-header-icon" />
+          {children}
+          {actions
+            ? actions
+                .filter((action): action is NonNullable<typeof action> =>
+                  Boolean(action),
+                )
+                .map(({ key, label, leftIcon, rightIcon, onClick }) => (
+                  <Button
+                    key={key}
+                    onClick={onClick}
+                    leftIcon={leftIcon ? <Svg src={leftIcon} /> : undefined}
+                    rightIcon={rightIcon ? <Svg src={rightIcon} /> : undefined}
+                    variant="ghost"
+                  >
```

Comment:
Ce serait pas mal de laisser les devs pouvoir surcharger d'autres props que simplement les 5 misent initialement non ? (pareil pour l'autre bouton aussi du coup)
```suggestion
                .map(({ label, leftIcon, rightIcon }) => (
                  <Button
                    leftIcon={leftIcon ? <Svg src={leftIcon} /> : undefined}
                    rightIcon={rightIcon ? <Svg src={rightIcon} /> : undefined}
                    variant="ghost"
                    {...props}
                  >
```

Reply:
Oui je serais plus à KISS et voir au besoin. On a bcp itéré avec @adrien-dos sur le besoin notamment pour ajouter les icones etc et je pense qu'on couvre pas mal là, je resolve si ca te va dans ce sens